src/inputs/forms/new-json-schema-form/components/GenerateIdFormat.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div class="generate-id-format">
+        <p-button style-type="gray-border" :outline="true"
+                  class="generate-button"
+                  @click="handleClickGenerate"
+        >
+            Generate
```

Comment:
Is it okay if there is no translation for this part?

Reply:
I added it :) Thx!